src/Tasks/Microsoft.NET.Build.Tasks/ConflictResolution/FileUtilities.netstandard.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#if NETCOREAPP1_0
+
+using System;
+using System.IO;
+using System.Reflection.Metadata;
+using System.Reflection.PortableExecutable;
+
+namespace Microsoft.NET.Build.Tasks.ConflictResolution
+{
+    static partial class FileUtilities
+    {
+        private static Version GetAssemblyVersion(string sourcePath)
```

Comment:
Depending on what you're targeting in your task DLL you may be able to remove this cross-compilation.  AssemblyName.GetAssemblyName was added back in netstandard2.0.